frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx

```diff
@@ -66,11 +66,8 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
       },
     };
 
-    const containers: ContainerSpec[] = _.get(
-      requestDeployment,
-      'spec.template.spec.containers',
-      [],
-    );
+    const containers: ContainerSpec[] =
+      requestDeployment?.[0]?.spec?.template?.spec?.containers ?? [];
```

Comment:
Why [0] ?

Reply:
Again requestDeployment doesn't have a property spec

Reply:
@afreen23 Same for this too. requestDeployment doesn't have property spec

Reply:
@Jayashree-panda this will break the existing functionality.
Please  fix the types wherever required